packages/wallet-connect/src/wallet-connect.ts

```diff
@@ -116,13 +116,15 @@ class WalletConnectWallet implements BaseWallet {
 export class WalletConnectProvider implements BaseWalletProvider {
   config: WalletConnectConfiguration;
   appName: string;
+  chainId?: PolkadotNamespaceChainId;
 
-  constructor(config: WalletConnectConfiguration, appName: string) {
+  constructor(config: WalletConnectConfiguration, appName: string, chainId?: PolkadotNamespaceChainId) {
```

Comment:
Any reason this is passed as a separate parameter, and not as part of config?! can you move chainId to config and make it an array.
walletConnect client takes chains as an array of chainIds, and works with multiple chains.

Reply:
Sorry, took me quite a while to get back to this, I changed it so that it's passed in the config and I also did some refactoring to account for multiple chainIds.

Reply:
Thank you!